core/src/main/kotlin/org/evomaster/core/output/TestCaseWriter.kt

```diff
@@ -509,6 +518,23 @@ class TestCaseWriter {
         //handleExpectations(res, lines, true)
     }
 
+    private fun addObjectAssertions(resContents: LinkedTreeMap<*,*>, lines: Lines){
+        resContents.keys
+                .filter{ !(it as String).contains("timestamp")}
```

Comment:
are u assuming the key is always a String?
could also add a comment specifying we want to avoid assertion on time-based fields, as those would lead to flaky tests

Reply:
For the moment, I am making that assumption, yes. The solution here is a bit of a hack to get it up and running. A more permanent (and hopefully elegant) solution would be to handle this via the flaky test handling process (when that is ready).

I have added a comment to this effect as well.